### ISSUE TITLE
Explaining how to restart CircleCI

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/conda-forge.github.io#595]
Restarting builds on CircleCI is a two step process that is poorly documented. We may consider including this in our docs. Have done this through screenshots below.

Step 1: Select the workflow of the build

<img width="1552" alt="screen shot 2018-06-02 at 00 37 42" src="https://user-images.githubusercontent.com/3019665/40872090-4a546482-65fd-11e8-8734-aa0b0cbc698e.png">

Step 2: Restart the workflow

<img width="1552" alt="screen shot 2018-06-02 at 00 39 51" src="https://user-images.githubusercontent.com/3019665/40872105-77e14834-65fd-11e8-8fb0-335f0a0a5e0e.png">

**Comment by mbargull** [comment]
Also, xref https://github.com/conda-forge/staged-recipes/pull/5935#issuecomment-393979820:
>> git commit --allow-empty -m trigger && git push
> 
> It would be really nice to put that in the conda-forge docs

For cases were the contributor does not have permissions to do restart the CI, they could push an empty commit instead, reissuing all new CI jobs.

**Comment by insertinterestingnamehere** [comment]
Another alternative that often applies: amending the previous commit (without any changes) and then force pushing it also restarts the build.

**Comment by jakirkham** [comment]
Certainly works. Get a little leary of this when collaborating with others on a PR.

**Comment by mbargull** [comment]
I'd highly **discourage** people to amend + force push for this. Force pushing breaks references (not only in the Git repo itself but also on other GitHub things) and can easily make valuable (review) comments *"go poof"*. It is a tool that should be used very sparingly and only when the consequences have been carefully considered, IMO.
<sub>There are legitimate cases where it is "okay" to force push on a PR branch, yes. But I'll gladly oppose proposals to put something like "BTW, it's okay to force push" into any docs :wink:...</sub>

**Comment by insertinterestingnamehere** [comment]
Yes, amend/force push should be used judiciously. In this context it's very helpful for avoiding the streams of nonsensical commits that might otherwise show up when debugging CI issues. When trying various fixes you can either send out a series of mostly meaningless commits or you can override a single commit until it's actually correct. There are trade offs. I prefer the cleaner history, but that's just me. Perhaps the best thing to tell beginners is to close/reopen their pull requests. That generates more e-mail noise and doesn't help them with debugging CI issues without muddying their commit history, but it does successfully restart the builds.
